src/utils/viewer/viewer.cpp

```diff
@@ -533,6 +536,62 @@ Viewer::Viewer(const Arguments& arguments)
   cameraSensorSpec->resolution = esp::vec2i(viewportSize[1], viewportSize[0]);
   agentConfig.sensorSpecifications = {cameraSensorSpec};
 
+  // add the new fisheye sensor
```

Comment:
Could you wrap all this code in a helper function, to keep viewer.cpp a bit more readable?

Reply:
OK. will do.